packages/miniflare/src/plugins/kv/index.ts

```diff
@@ -41,6 +41,7 @@ export const KVOptionsSchema = z.object({
 	assetsPath: PathSchema.optional(),
 	assetsKVBindingName: z.string().optional(),
 	assetsManifestBindingName: z.string().optional(),
+	assetsReverseMapBindingName: z.string().optional(),
```

Comment:
Not needed?

Reply:
agreed. This is an internal implementation detail

Reply:
oops I meant to remove that